spec/requests/portfolios_rbac_read_access_spec.rb

```diff
@@ -32,6 +32,21 @@
       end
     end
 
+    context "Catalog User should not see any portfolios by default" do
+      before do
+        allow(Insights::API::Common::RBAC::Access).to receive(:new).with('portfolios', 'read').and_return(access_obj)
+        allow(Insights::API::Common::RBAC::Roles).to receive(:assigned_role?).with(catalog_admin_role).and_return(false)
```

Comment:
If `catalog_admin_role` is no longer supported by RBAC, should we be rename this to `org_admin`?

Reply:
@syncrou Catalog Admin Role is still there it has not been removed.